tests/v2/validation/provisioning/k3s/custom_cluster_test.go

```diff
@@ -159,6 +163,15 @@ func (c *CustomClusterProvisioningTestSuite) ProvisioningK3SCustomCluster(extern
 				clusterToken, err := clusters.CheckServiceAccountTokenSecret(client, clusterName)
 				require.NoError(c.T(), err)
 				assert.NotEmpty(c.T(), clusterToken)
+
+				err = hardening.CheckClusterHardening(client, tt.hardening.Hardened, nodes, tt.nodeRoles)
+				require.NoError(c.T(), err)
+
+				hardenCluster := clusters.HardenK3SRKE2ClusterConfig(clusterName, namespace, "", "", kubeVersion, nil)
```

Comment:
I'm curious as to how / why this doesn't run when hardened = false 🤔

Reply:
In the other 2.6 PR, Israel had the same concern as we discussed offline, so I'll definitely take another look into this